modules/ROOT/pages/java-embedded/indexes.adoc

```diff
@@ -4,10 +4,10 @@
 [[java-embedded-new-index]]
 = Using indexes
 
-It is possible to create and use all the index types described in xref:4.0@cypher-manual:ROOT:indexes-for-search-performance/index.adoc[Cypher Manual -> Indexes].
+It is possible to create and use all the index types described in xref:4.0@cypher-manual:ROOT:administration/indexes-for-search-performance/index.adoc[Cypher Manual -> Indexes for search performance].
```

Comment:
Thanks this is valid for versions:

4.0 - https://neo4j.com/docs/cypher-manual/4.0/administration/indexes-for-search-performance/
4.1 - https://neo4j.com/docs/cypher-manual/4.1/administration/indexes-for-search-performance/
4.2 - https://neo4j.com/docs/cypher-manual/4.2/administration/indexes-for-search-performance/